R/axis.R

```diff
@@ -38,4 +38,89 @@ easy_rotate_x_labels <- function(angle = 90, side = c("left", "middle", "right")
     ggplot2::theme(axis.text.x = ggplot2::element_text(angle = angle, hjust = hjust))
 }
 
+#' Easily remove axe(s)
```

Comment:
`Easily remove one or more axes`?

Reply:
yeah that'd be better.  no such thing as an axe! in a typical plot, anyways